base_layer/wallet_ffi/scripts/prepare_distribution.py

```
@@ -0,0 +1,308 @@
+    
+    # Create core package structure
+    print("Creating core package...")
+    core_dir = base_dir / "tari_wallet_core"
+    core_dir.mkdir(exist_ok=True)
+    
+    # Copy core module
+    shutil.copytree("python/tari_wallet_core", core_dir, dirs_exist_ok=True)
+    
+    print("  ✓ Created core package")
+    
+    print(f"\nPackage preparation complete!")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)